image_client/client_tools.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env python3
 import argparse
+import botany_importer_config
```

Comment:
update requirements.txt to include new modules? probably not necessary for these imports, but it's overdue, so may as well get it now.